tasks/generate.yml

```diff
@@ -0,0 +1,42 @@
+---
+
+- block:
+    - name: Create ssh key generation directory
+      file:
+        path: "{{ item.path | dirname }}"
+        owner: "{{ item.owner }}"
+        group: "{{ item.group | default(item.owner) }}"
+        state: directory
+      with_items: "{{ ssh_keys_generate_keys }}"
+
+    - name: Generate private ssh key
+      shell:
+        "{{ ssh_keys_generate_command | default(_ssh_keys_generate_command) }}"
+      tags:
+        - skip_ansible_lint
+      when: >-
+        item.force | default(false) | bool or
+        item.path is not file
+      with_items: "{{ ssh_keys_generate_keys }}"
+
+    - name: Set permissions on keys
+      file:
+        path: "{{ item.path }}"
+        owner: "{{ item.owner }}"
+        group: "{{ item.group | default(item.owner) }}"
+        mode: "{{ item.mode | default('0600') }}"
+      with_items: "{{ ssh_keys_generate_keys }}"
+
+    - name: Generate public ssh key
+      openssl_publickey:
+        path: "{{ item.path }}.pub"
+        force: "{{ item.force | default(false) }}"
+        format: "{{ item.format | default('OpenSSH') }}"
+        # comment: "{{ item.comment | default(omit) }}"
```

Comment:
No `comment`?

Reply:
I have forked the `openssl_publickey` module to add a `comment` key. I removed the commented line.